src/Epic/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Though in general I think it's good to split code up into separate modules, I wonder if in this case it might be slightly misleading. I think in future we/someone else might look at this file and assume (given that it's a regular `ts` file under `src`) that it makes it into the final bundle for `@guardian/braze-components` (though I don't think it will since the only place which `import`s it is the story). Maybe inlining this in the story would remove that ambiguity, what do you reckon?

Reply:
I agree, they're combined now.